frontend/src/metabase/visualizations/components/ScalarValue/utils.unit.spec.ts

```diff
@@ -44,14 +45,15 @@ describe("findSize", () => {
 
   it("returns the first size with which text width is smaller than the target width", () => {
     measureTextSpy
-      .mockImplementationOnce(createMockMeasureText(120))
-      .mockImplementationOnce(createMockMeasureText(110))
-      .mockImplementationOnce(createMockMeasureText(100)) // this is the one we want
-      .mockImplementationOnce(createMockMeasureText(90));
+      .mockImplementationOnce(createMockMeasureText(120, 50))
+      .mockImplementationOnce(createMockMeasureText(110, 50))
+      .mockImplementationOnce(createMockMeasureText(100, 50)) // this is the one we want
+      .mockImplementationOnce(createMockMeasureText(90, 50));
```

Comment:
nit: I think to somewhat show the actual behavior you should shrink the height proportionally to the width, since `findSize` will use the same text but with reduced `size` each time. So you could get a result that will be narrower and shorter each time.